src/backend/access/external/pxffilters.c

```diff
@@ -376,6 +467,46 @@ pxf_serialize_filter_list(List *expressionItems)
 
 		switch (tag)
 		{
+			case T_Var:
```

Comment:
Which scenario is the expr type is T_Var?
In you comment is IN(single_value),but IN(single_value) is also T_OpExpr(Var + Const)